tests/test_composable.py

```diff
@@ -67,11 +65,18 @@ def test_composable_valid(
     with open(current_config_path) as f:
         current_config = f.read()
 
-    # expecting output config has `use xxx from yyy`
-    config_name = test_config_file.split("/")[-1].removesuffix(".c4m")
-    config_path = "/".join((CONFIGS / test_config_file).__str__().split("/")[:-1])
-    use_output = f'use {config_name} from "{config_path}"'
-    assert use_output in current_config
+    if replace:
+        # replaces current config with content of incoming
+        with open(CONFIGS / test_config_file) as f:
+            original_config = f.read()
+            assert current_config == original_config
```

Comment:
```suggestion
        original_config = (CONFIGS / test_config_file).read_text()
        assert current_config == original_config
```

pathlib has useful abstractions around IO bits. lets use them whenever possible. similar above for already committed code